test.js

```diff
@@ -14,34 +14,34 @@ const type = 'log'
 const consistency = 'one'
 const host = 'localhost'
 const port = 9200
+// new data in Elasticsearch is only visible after the refreshInterval
+const refreshInterval = 1100 // assume ES default settings 1 second
```

Comment:
No :( - I think 1s is the minimum setting (and on my machine the test failed sometimes). 
From Elasticsearch docs: 
"index.refresh_interval
How often to perform a refresh operation, which makes recent changes to the index visible to search. Defaults to 1s. Can be set to -1 to disable refresh."

In production people use often 5 - 10s or disable refresh temporary to make indexing faster. 
https://sematext.com/blog/2013/07/08/elasticsearch-refresh-interval-vs-indexing-performance/